File: src/integration-tests/expecteds/py/additional-props.py
from typing import NewType
from typing import Union
from typing import Any
from typing import Mapping

D = NewType("D", str)

C = NewType("C", str)

A = NewType("A", Mapping[Any, Any])

B = NewType("B", Mapping[Any, Any])
"""Generated! Represents an alias to any of the provided schemas
"""
AnyOfAB = NewType("AnyOfAB", Union[A, B])

File: src/integration-tests/expecteds/ts/additional-props.ts
export type D = string;
export type C = string;
export interface A { [key: string]: any; }
export interface B {
  c?: C;
  [k: string]: any;
}
/**
 *
 * Generated! Represents an alias to any of the provided schemas
 *
 */
export type AnyOfAB = A | B;
